cougar-framework/cougar-client/src/main/java/com/betfair/cougar/client/SocketContextEmitter.java

```
@@ -0,0 +1,63 @@
+package com.betfair.cougar.client;
+
+import com.betfair.cougar.api.RequestUUID;
+import com.betfair.cougar.api.UUIDGenerator;
+import com.betfair.cougar.api.geolocation.GeoLocationDetails;
+import com.betfair.cougar.client.api.ContextEmitter;
+import com.betfair.cougar.client.api.GeoLocationSerializer;
+import org.apache.http.Header;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Standard context emitter for use with socket client transports (based on HttpContextEmitter)
+ */
+public class SocketContextEmitter<C> implements ContextEmitter<Map<String, String>, C> {
```

Thanks. I wasn't aware of this (and failed to read the code instead of assuming it was implemented like http). Fixed.